extension/intents/tabs/tabs.js

```diff
@@ -606,10 +606,20 @@ intentRunner.registerIntent({
     const hiddenTabs = await browser.tabs.query({ hidden: true });
     const numOfOpenTabs = tabs.length - hiddenTabs.length;
     const tabsOpen = numOfOpenTabs <= 1 ? "Open tab" : "Open tabs";
+    const sampleTabs = await browser.windows.getAll({ populate: true });
+    let tabsCount = 0;
+    let windowsCount = 1;
+    sampleTabs.forEach((window, index) => {
+      tabsCount += window.tabs.length ;
+      windowsCount += index;
+    });
 
     const card = {
       answer: {
-        largeText: `${numOfOpenTabs}`,
+        largeText: `Open Windows: ${windowsCount} ,
```

Comment:
I think if you switch this to `text:` instead of `largeText:` that this will display better.

Reply:
The component rendering answers is referencing `largeText` as output
```
 <div className="results-text">
        <em>
          {card.answer.largeText ? (
            <div className="results-largeText">{card.answer.largeText}</div>
          ) : null}
          <div>{card.answer.text}</div>
        </em>
      </div>
```
I could change the CSS class to `results-text`

Reply:
Since `largeText` is being used by other outputs I created a function to handle the text based on which tab method is called